src/plugins/intel_gpu/tests/test_cases/resample_gpu_test.cpp

```diff
@@ -2235,11 +2233,20 @@ INSTANTIATE_TEST_SUITE_P(resample_opt_smoke_nearest,
                             }
                         ));
 
-INSTANTIATE_TEST_SUITE_P(resample_opt_smoke_linear_onnx,
+INSTANTIATE_TEST_SUITE_P(resample_opt_smoke_linear_onnx_4d_padding,
+                         resample_opt_random_test,
```

Comment:
Is this new kernel covered by fusion unit tests? If not, please add couple of test cases

Reply:
And several functional test cases would be good too.

Reply:
agree, added